core/src/test/java/org/infinispan/util/CoreTestBlockHoundIntegration.java

```diff
@@ -67,6 +73,10 @@ private static void allowTestsToBlock(BlockHound.Builder builder) {
       CommonsBlockHoundIntegration.allowPublicMethodsToBlock(builder, NotifierLatch.class);
 
       CommonsBlockHoundIntegration.allowPublicMethodsToBlock(builder, TestBlocking.class);
+
+      CommonsBlockHoundIntegration.allowMethodsToBlock(builder, Class.forName(ReplListener.class.getName() + "$ReplListenerInterceptor"), false);
```

Comment:
Assuming this is just for the `Thread.sleep()` call, I think it would be better to add an executor parameter to `TestingUtil.delayed()` and to inject the non-blocking executor in `ReplListenerInterceptor`.

Reply:
Actually am not sure why I didn't just add `TestingUtil#sleepThread` to the exception list. Let me try that instead.

Reply:
Ah it was because of `logCommand` that acquires a lock. I think I will leave it as is for now.

Reply:
But I can also add `TestingUtil#sleepThread` as okay to block though too.

Reply:
+1 to add `TestingUtil#sleepThread`

Reply:
Seems to work fine without the `ReplListenerInterceptor` exception now.